src/guide/11-fgpu2-source/1-build-from-source.rst

```diff
@@ -1,2 +1,4 @@
 Building FGPU2 from Source
 ==========================
+
+**TODO: Do we want to duplicate readme.md here or just link to it?**
```

Comment:
Duplicate or atleast summarise would be my thoughts. Rather than externally linking from things that should be core guide. Probably easier to include some screenshots in the guide.

Reply:
Arguably, the readme should be updated to link to docs quite prominently.